src/Bicep.LangServer/Handlers/ImportKubernetesManifestHandler.cs

```diff
@@ -150,6 +151,7 @@ private static ResourceDeclarationSyntax ProcessResourceYaml(YamlDocument yamlDo
                 SyntaxFactory.CreateStringLiteral($"{type}@{apiVersion}"),
                 null,
                 SyntaxFactory.AssignmentToken,
+                ImmutableArray<Token>.Empty,
```

Comment:
Is this for the newlines? If so, how come there is only one and not two since there OpenNewlines and CloseNewlines properties

Reply:
This is for resource declaration with `if-expression`, which can only have newlines after `=`.